newIDE/app/src/ObjectEditor/Editors/TextEditor.js

```diff
@@ -97,26 +107,64 @@ export default class TextEditor extends React.Component<EditorProps, void> {
             }}
             style={styles.checkbox}
           />
-          <MiniToolbarText>
-            <Trans>Font:</Trans>
-          </MiniToolbarText>
-          <ResourceSelector
-            margin="none"
-            project={project}
-            resourceManagementProps={resourceManagementProps}
-            resourcesLoader={ResourcesLoader}
-            resourceKind="font"
-            fullWidth
-            canBeReset
-            initialResourceName={textObjectConfiguration.getFontName()}
-            onChange={resourceName => {
-              textObjectConfiguration.setFontName(resourceName);
-              this.forceUpdate();
-            }}
-            hintText={<Trans>Choose a font</Trans>}
-            style={styles.resourcesSelector}
-          />
+          <ButtonGroup>
+            <Tooltip title={<Trans>Align text on the left</Trans>}>
```

Comment:
I think you can use the component IconButton with the `tooltip` prop

Reply:
You might need to add some props to the component

Reply:
I used the same way as `YesNoField`. It doesn't seem possible to do the color inversion with `IconButton`.
I added a screenshot of the UI at the end of the description.

Reply:
I'm fine with this exceptional use of material ui components because this is the first time we're doing a "three part toggle". If we do it again, we'll factor it.